File: python-code/ex1/carro.py
class Carro:
  def __init__(self, marca, modelo, ano):
    self.marca = marca
    self.modelo = modelo
    self.ano = ano

  def exibir_detalhes(self):
    print(f"Marca: {self.marca}, Modelo: {self.modelo}, Ano: {self.ano}")

carro1 = Carro("Renault", "Sandero", 2018)
carro2 = Carro("volkswagen ", "Gol", 2015)
carro3 = Carro("Jeep", "Compass", 2023)

carro1.exibir_detalhes()
carro2.exibir_detalhes()
carro3.exibir_detalhes()

File: java-code/src/main/java/ex1/Carro.java
package ex1;

public class Carro {
    String marca;
    String modelo;
    int ano;

    Carro(String marca, String modelo, int ano) {
        this.marca = marca;
        this.modelo = modelo;
        this.ano = ano;
    }

    void exibirDetalhes() {
        System.out.println("Marca: " + this.marca + ", Modelo: " + this.modelo + ", Ano: " + this.ano);
    }
}
